forms-shared/src/definitions/devFormDefinitions.ts

```diff
@@ -21,6 +26,29 @@ export const devFormDefinitions: FormDefinition[] = [
       ginisPersonName: '',
     },
     isSigned: false,
+    sharepointData: {
```

Comment:
I see beneficial to move all of this to `forms-shared/src/sharepoint/mappings/ziadost-o-pridelenie-najomneho-bytu.ts` and just use `sharepointData: ziadostOPridelenieNajomnehoBytuSharepointData` here.